rebasehelper/cli.py

```diff
@@ -103,8 +104,32 @@ def __getattr__(self, name):
 
 class CliHelper:
 
-    @staticmethod
-    def run():
+    @classmethod
+    def convert_macros_to_dict(cls, macros_list: List[str]) -> Dict[str, str]:
+        """Converts macros from CLI to a dictionary.
+
+        Args:
+            macros_list: List of macros in the format 'MACRO EXPRESSION'.
+
+        Returns:
+            The converted macros, MACRO are keys, EXPRESSION are values.
+
+        Raises:
+            RebaseHelperError if the macros don't follow the correct format.
+
+        """
+        macros = {}
+        for macro in macros_list:
+            match = re.match(r"\s*%?(?P<macro>\S+)\s+(?P<expr>\w+(\s*\w+)*)", macro)
```

Comment:
This is wrong. Macro value can be anything, not just word characters and whitespaces. This won't even match a simple path containing directory separators.